src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -922,7 +936,27 @@ ValidatorList::updateTrusted(hash_set<NodeID> const& seenValidators)
         << trustedMasterKeys_.size() << "  of " << keyListings_.size()
         << " listed validators eligible for inclusion in the trusted set";
 
-    quorum_ = calculateQuorum(trustedMasterKeys_.size(), seenValidators.size());
+    auto numUNL = trustedMasterKeys_.size();
+    auto numEffectiveUNL = numUNL;
```

Comment:
Nit: Unless I am misunderstanding, I think these would be better named something like unlSize, and effectiveUnlSize. nunUNL makes me think this variable represents the number of UNLs, as opposed to the size of the UNL.

Reply:
right, changed